jsk_arc2017_baxter/euslisp/lib/util.l

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env roseus
+
+(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/util.l")
+
+(ros::load-ros-manifest "jsk_gui_msgs")
+(ros::load-ros-manifest "jsk_recognition_msgs")
+
+(defun opposite-arm (arm) (if (eq arm :larm) :rarm :larm))
```

Comment:
When i use `(in-package ~~~)` in `apc-interface.l`, I can run `(jsk_arc2017_baxter::apc-init)` but `*ri*` was not set.
`in-package` shows really weird effect, and I do not want to use it.
Also, there is ns problem reported by @pazeshun.